### ISSUE TITLE
Multisite Subdir / Combine Issue

### ISSUE THREAD

**Issue by akincansenol** [futtta/autoptimize#38]
Hi @futtta 

Main site has been combine well but same domain subdir multisite (same theme, same plugins, same settings) hasn't properly combine / just one css combine file & js combine file but other css & js still viewing in code.

Piece of head output;

`
<!DOCTYPE html><html lang="en-GB" prefix="og: http://ogp.me/ns# fb: http://ogp.me/ns/fb#" class="no-js"><head><link type="text/css" media="all" href="https://cdn.example.com/en/wp-content/cache/autoptimize/2/css/autoptimize_d29859b170d88126b5eb8ef769bc04cc.css" rel="stylesheet" /><title>Home &laquo; Example - EN</title><meta http-equiv="x-dns-prefetch-control" content="on"><link rel="dns-prefetch" href="//ajax.googleapis.com"><link rel="dns-prefetch" href="//www.google.com"><link rel="dns-prefetch" href="//maps.googleapis.com"><link rel="dns-prefetch" href="//fonts.googleapis.com"><link rel="canonical" href="https://example.com/en/" />  <script type="text/javascript">window._wpemojiSettings = {"baseUrl":"https:\/\/s.w.org\/images\/core\/emoji\/72x72\/","ext":".png","source":{"concatemoji":"https:\/\/example.com\/en\/wp-includes\/js\/wp-emoji-release.min.js"}};
			!function(a,b,c){function d(a){var c,d=b.createElement("canvas"),e=d.getContext&&d.getContext("2d"),f=String.fromCharCode;return e&&e.fillText?(e.textBaseline="top",e.font="600 32px Arial","flag"===a?(e.fillText(f(55356,56806,55356,56826),0,0),d.toDataURL().length>3e3):"diversity"===a?(e.fillText(f(55356,57221),0,0),c=e.getImageData(16,16,1,1).data.toString(),e.fillText(f(55356,57221,55356,57343),0,0),c!==e.getImageData(16,16,1,1).data.toString()):("simple"===a?e.fillText(f(55357,56835),0,0):e.fillText(f(55356,57135),0,0),0!==e.getImageData(16,16,1,1).data[0])):!1}function e(a){var c=b.createElement("script");c.src=a,c.type="text/javascript",b.getElementsByTagName("head")[0].appendChild(c)}var f,g;c.supports={simple:d("simple"),flag:d("flag"),unicode8:d("unicode8"),diversity:d("diversity")},c.DOMReady=!1,c.readyCallback=function(){c.DOMReady=!0},c.supports.simple&&c.supports.flag&&c.supports.unicode8&&c.supports.diversity||(g=function(){c.readyCallback()},b.addEventListener?(b.addEventListener("DOMContentLoaded",g,!1),a.addEventListener("load",g,!1)):(a.attachEvent("onload",g),b.attachEvent("onreadystatechange",function(){"complete"===b.readyState&&c.readyCallback()})),f=c.source||{},f.concatemoji?e(f.concatemoji):f.wpemoji&&f.twemoji&&(e(f.twemoji),e(f.wpemoji)))}(window,document,window._wpemojiSettings);</script> <style type="text/css">img.wp-smiley,
img.emoji {
	display: inline !important;
	border: none !important;
	box-shadow: none !important;
	height: 1em !important;
	width: 1em !important;
	margin: 0 .07em !important;
	vertical-align: -0.1em !important;
	background: none !important;
	padding: 0 !important;
}</style><link rel='stylesheet' id='cmb-google-maps-css'  href='https://example.com/plugins/inventor/libraries/cmb_field_map/css/style.css' type='text/css' media='all' /><link rel='stylesheet' id='cmb-google-street-view-css'  href='https://example.com/plugins/inventor/libraries/cmb_field_street_view/css/style.css' type='text/css' media='all' /><link rel='stylesheet' id='inventor-poi-css'  href='https://example.com/plugins/inventor/assets/fonts/inventor-poi/style.css' type='text/css' media='all' /><link rel='stylesheet' id='contact-form-7-css'  href='https://example.com/plugins/contact-form-7/includes/css/styles.css' type='text/css' media='all' /><link rel='stylesheet' id='inventor-partners-css'  href='https://example.com/plugins/inventor-partners/assets/style.css' type='text/css' media='all' /><link rel='stylesheet' id='inventor-pricing-css'  href='https://example.com/plugins/inventor-pricing/assets/style.css' type='text/css' media='all' /><link rel='stylesheet' id='inventor-reviews-css'  href='https://example.com/plugins/inventor-reviews/assets/style.css' type='text/css' media='all' /><link rel='stylesheet' id='dashicons-css'  href='https://example.com/en/wp-includes/css/dashicons.min.css' type='text/css' media='all' /><link rel='stylesheet' id='wsl-widget-css'  href='https://example.com/plugins/wordpress-social-login/assets/css/style.css' type='text/css' media='all' /><link rel='stylesheet' id='essb_theme_standard-css'  href='https://example.com/plugins/easy-social-share-buttons3/lib/modules/top-posts-widget/css/essb-theme-standard.min.css' type='text/css' media='all' /><link rel='stylesheet' id='essb-font-css'  href='https://example.com/plugins/easy-social-share-buttons3/lib/modules/top-posts-widget/css/essb-posts-font.css' type='text/css' media='all' /><script type='text/javascript' src='//ajax.googleapis.com/ajax/libs/jquery/1.11.3/jquery.min.js'></script> <script type='text/javascript'>try{jQuery.noConflict();}catch(e){};</script> <script type='text/javascript' src='https://example.com/plugins/ultimate-tweaker/sections/content/content_link_target_blank/post-link-target-blank.min.js'></script> <script type='text/javascript' src='https://example.com/plugins/inventor/libraries/cmb_field_map/js/script.js'></script> <script type='text/javascript' src='https://example.com/plugins/inventor/libraries/cmb_field_street_view/js/script.js'></script> <script type='text/javascript' src='//maps.googleapis.com/maps/api/js?key=XXXXXXXXXXXXXXXXXXXXXXXXXXX;libraries=weather%2Cgeometry%2Cvisualization%2Cplaces%2Cdrawing'></script> <script type='text/javascript'>/*  */
var userSettings = {"url":"\/","uid":"161028","time":"1458575218","secure":"1"};
/*  */</script> <script type='text/javascript' src='https://example.com/en/wp-includes/js/plupload/plupload.full.min.js'></script>  <script type='text/javascript' src='//ajax.googleapis.com/ajax/libs/jqueryui/1.11.4/jquery-ui.min.js'></script> <link rel='https://api.w.org/' href='https://example.com/en/wp-json/' /><link rel="EditURI" type="application/rsd+xml" title="RSD" href="https://example.com/en/xmlrpc.php?rsd" /><link rel='shortlink' href='https://example.com/en/' /><meta name="generator" content="Custom Login v3.2.4" /><style>.getty.alignleft {
			margin-right: 5px;
		}
		.getty.alignright {
			margin-left: 5px;
		}</style><link rel="alternate" hreflang="tr-TR" href="https://example.com/?noredirect=tr_TR" /><link rel="alternate" hreflang="en-GB" href="https://example.com/en/" /><link rel="apple-touch-icon" href="https://example.com/wp-content/uploads/2016/03/iphone60.png" /><link rel="apple-touch-icon" sizes="76x76" href="https://example.com/wp-content/uploads/2016/03/ipad76.png" /><meta name="copyright" content="All rights reserved © Example, 2016"> <script type="text/javascript">var essb_settings = {"ajax_url":"https:\/\/example.com\/en\/wp-admin\/admin-ajax.php","essb3_nonce":"f185240593","essb3_plugin_url":"https:\/\/example.com\/plugins\/easy-social-share-buttons3","essb3_facebook_total":true,"essb3_admin_ajax":true,"essb3_internal_counter":false,"essb3_stats":true,"essb3_ga":true,"essb3_ga_mode":"extended","essb3_counter_button_min":50,"essb3_counter_total_min":500,"blog_url":"https:\/\/example.com\/en\/","ajax_type":"light","essb3_postfloat_stay":false,"essb3_no_counter_mailprint":true,"essb3_single_ajax":true,"twitter_counter":"api","post_id":521};</script><link rel="icon" href="https://example.com/en/wp-content/uploads/sites/2/2016/03/cropped-logo512-32x32.png" sizes="32x32" /><link rel="icon" href="https://example.com/en/wp-content/uploads/sites/2/2016/03/cropped-logo512-192x192.png" sizes="192x192" /><link rel="apple-touch-icon-precomposed" href="https://example.com/en/wp-content/uploads/sites/2/2016/03/cropped-logo512-180x180.png" /><meta name="msapplication-TileImage" content="https://example.com/en/wp-content/uploads/sites/2/2016/03/cropped-logo512-270x270.png" /></head>`

**Comment by futtta** [comment]
are the two sites on the same (sub-)domain?

**Comment by akincansenol** [comment]
No, example.com and example.com/en by the way I found the issue, that comes with,

define("WP_PLUGIN_URL", "https://example.com/plugins");

So it's not compatible with this I guess, also I found your reply on WP Support topic

> reason is folder structure being different in some cases, cfr.
/eshop-MRBUTTON/application/eshopbox-drop-down-cart-widget/widget-flyout-cart-style.css?ver=4.4.2 -> aggregated by AO
/eshopbox/eshop-includes/js/jquery/jquery.js?ver=1.11.3 -> not aggregated by AO

this might work better with the upcoming AO 2.0.1 (out end feb/ beginning march) which has this fix included. you could try appliying that fix or download the entire 2.0.1-to-be to see if that fixes your issue.

hope this helps,
frank

**Comment by futtta** [comment]
if your plugins-folders are located outside of wp-content, then indeed it is not compatible I'm afraid.

**Comment by akincansenol** [comment]
Thank you for information @futtta, you very fast!
